src/hdmf/container.py

```diff
@@ -759,7 +745,26 @@ def _func(self, **kwargs):
         return _func
 
     @classmethod
-    def __make_setter(cls, nwbfield, add_name):
+    def __make_getter(cls, attr):
```

Comment:
Just to make sure I understand/remember correctly.  We need this to set the parent/child stuff on the multiple containers in this MCI. i.e. we can't use Container.__make_setter (or whatever it's called) because it's just not aware of the multiple containers being in a LabelledDict.  

Is that correct?

Reply:
We can't use __make_setter because other code assumes that `getattr(self, attr_name)` exists and is a LabelledDict. For example in the add, get, and getitem methods (and add/get are called by many things). I could move the initialization of `fields['attr']` there.

Reply:
I think it's fine the way it is. I was just trying to understand what was going on.